Sources/OpenTelemetrySdk/Internal/MillisClock.swift

```diff
@@ -23,10 +23,10 @@ public class MillisClock: Clock {
     }
 
     public var now: UInt64 {
```

Comment:
should this class be public? Also why wouldn't SDK operate with `Date` and only convert to nanos on export?

Reply:
Yes, probably shouldnt be public, I will change. 
About using directly `Date` I remember having doubt about it at the beggining of the project, in fact there are some functions that support `Date` for users, but don't remember why I decided to use nanos at the end. I can revisit it in a future. I will create an issue with it